src/frame.rs

```diff
@@ -178,10 +249,12 @@ impl FrameHeader {
         }
     }
 
+    /// Read the dictionary ID required to decode the frame from the header.
```

Comment:
Just "the dictionary ID required to decode the frame from the header.
Comment"?

Reply:
I reworded it in a way that's more concise, let me know if you'd prefer your wording.